wordpress-mobile/WordPress-iOS
# Jetpack App (Basics): Update notifications dot color

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/16356

This PR updates the icon-tab-notifications-unread for the Jetpack target

![Simulator Screen Shot - iPhone 12 Pro - 2021-04-27 at 12 18 35](https://user-images.githubusercontent.com/6711616/116182054-9ee89200-a756-11eb-8feb-13ec3249ab69.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-27 at 12 18 39](https://user-images.githubusercontent.com/6711616/116182061-a14aec00-a756-11eb-8c3d-1ba2c78f9993.png)
-- | --


## How to test
1. Delete the existing Jetpack app.
2. Build and run the Jetpack app.
3. Login with your credentials.
4. ✅ Notice that the notification dot is green instead of pink.

## Regression Notes
1. Potential unintended areas of impact
WordPress notification icon

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Built and tested both targets

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
